src/main/java/org/n52/wps/repository/git/GitAlgorithmRepository.java

```diff
@@ -89,60 +93,103 @@
 
     private String filenameRegex;
 
+    private String branchName;
+
     private Repository localRepo;
 
     private final Map<IAlgorithm, ProcessDescription> processDescriptions;
 
     private final Map<String, IAlgorithm> javaAlgorithms;
 
-//    private final Map<String, IAlgorithm> rAlgorithms;
-    
     private final Map<File, String> file2Wkn;
 
-    private ConfigurationModule gitAlgorithmRepoConfigModule;
+    private GitAlgorithmRepositoryCM configuration;
 
     private CustomClassLoader customClassLoader;
 
     private Collection<DiffEntry> changedFiles;
 
-    public GitAlgorithmRepository() throws UpdateGitAlgorithmsRepositoryException, GitAlgorithmsRepositoryConfigException {
```

Comment:
It seems that because the empty constructor was removed, the GitAlgorithm repository fails to load now.